core/src/apps/bitcoin/sign_tx/zcash.py

```diff
@@ -103,6 +94,8 @@ def init_hash143(self) -> None:
     def hash143_preimage_hash(
         self, txi: TxInputType, public_keys: List[bytes], threshold: int
     ) -> bytes:
+        ensure(self.tx.version == 4)
```

Comment:
i don't think this is necessary, see above

Reply:
Fixed 016af57.